app/models/service_template_ansible_playbook.rb

```diff
@@ -179,6 +178,12 @@ def retirement_potential?
 
   private
 
+  def retain_resource_actions
```

Comment:
@syncrou we need a better name for this method. It is no longer to retain resource actions.